src/apps/proxmox/ve/restapi/mode/vmusage.pm

```diff
@@ -310,30 +325,38 @@ __END__
 
 =head1 MODE
 
-Check VM usage on Proxmox VE Cluster.
+Check virtual machines usage on Proxmox VE Cluster.
 
 =over 8
 
 =item B<--vm-id>
 
-Exact VM ID.
+Exact virtual machine ID.
 
 =item B<--vm-name>
 
-Exact VM name (if multiple names: names separated by ':').
+Exact virtual machine name (if multiple names: names separated by ':').
 
 =item B<--use-name>
 
-Use VM name for perfdata and display.
+Use virtual machine name for perfdata and display.
+
+=item B<--include-node-name>
+
+Filter only virtual machine running on specified node name (can be a regexp).
+
+=item B<--exclude-name>
+
+Exclude by virtual machine name (can be a regexp).
 
```

Comment:
It is a bit unexpected to see this option detailed before --filter-name, which is equivalent to our new routine --include-name